packages/utils/src/logger.ts

```diff
@@ -1,11 +1,7 @@
 import { WrappedFunction } from '@sentry/types';
 
-import { IS_DEBUG_BUILD } from './flags';
 import { getGlobalObject, getGlobalSingleton } from './global';
 
-// TODO: Implement different loggers for different environments
-const global = getGlobalObject<Window | NodeJS.Global>();
```

Comment:
This caused node 8 tests to fail. Defining a variable "global" is probably not a good idea because of namespace collisions so I renamed it in some instances and moved it into local scope here.

Reply:
Given that's how we do it across the the SDK, I'd rather keep that consistent and just fix the tests. This, in theory, is the problem the const-to-var transformer is supposed to solve. Lemme take a look at the tests and see if I can figure out why that's not working.

Reply:
If it helps me getting this PR merged I'm happy to make this change in all of the SDK.

I genuinely believe declaring a variable called `global` is a bad idea. It would be the same as doing `const console = ...` or `const setTimeout = ...`. Feels weird doesn't it? I never like the argument "because we always did it that way so we must continue with it".

Also, I don't think the tests are at fault here. They're probably doing the right thing ,because they are now surfacing this. A transformer should IMO not be necessary for stuff like this. It's solving a problem that shouldn't even exist.

Reply:
> I genuinely believe declaring a variable called global is a bad idea. It would be the same as doing const console = ... or const setTimeout = .... Feels weird doesn't it? I never like the argument "because we always did it that way so we must continue with it".

As a general matter, I agree with you. In this case, though, it's not just "it's what we've always done." The reason we do it that way is that most of the time, the thing we're talking about _is_ `global`, and so calling it `global` is the simplest path for the reader. But we have to do the `getGlobalObject` dance, IIRC because we can't (or, at least, didn't used to be able to) use `globalThis`. Lemme see if that still applies...

UPDATE: It seems we took `globalThis` out only because we were [trying to support versions of TS and Angular we no longer support](https://github.com/getsentry/sentry-javascript/issues/2978). So I wonder if we could just replace `const global = getGlobalObject(); global.blahblahblah()` with `globalThis.blahblahblah()` and skirt the issue entirely. WDYT?

Reply:
Removed the changes to global from this PR. Tests seem to be working again. Gremlins.

Reply:
LOL. Great. I think the `global`/`globalThis` convo might be worth having, but I agree - separate PR for that.